src/relayers/relayersDataController.ts

```diff
@@ -21,14 +33,30 @@ export class RelayersController extends Controller {
     }
 
     /**
-     * Retrieves a full list of relayers, along with the unbounded sum SLA scores
+     * Retrieves a paged list of relayers, along with the unbounded sum SLA scores
      * after a given cutoff.
      * @param slaSince A UNIX timestamp starting from which the SLA score will be summed.
      **/
     @Get("")
     public async getRelayers(
+        @Query() page = 0,
+        @Query() perPage = 20,
+        @Query() sortBy: RelayerColumns = "block_number",
+        @Query() sortAsc = false,
+        @Query() slaSince: number
+    ): Promise<RelayerData[]> {
+        return getAllRelayers(page, perPage, sortBy, sortAsc, [], slaSince);
+    }
+
+    @Post("")
+    public async getFilteredRelayers(
+        @Query() page = 0,
+        @Query() perPage = 20,
+        @Query() sortBy: RelayerColumns = "block_number",
+        @Query() sortAsc = false,
+        @Body() filters: Filter<RelayerColumns>[] = [],
```

Comment:
What is the security implication of this, would it be possible to manipulate the query?

Reply:
Good catch actually. I relied on TS typechecking but that won't help at runtime. Fixing now